### PR TITLE
new exercise remove for objects

### PR DESCRIPTION
**Pull request by MSilva95**

### CODE REVIEW

**Review by kigiri — CHANGES_REQUESTED** [review]
I don't think removing from an object is ever a good idea in JS.
We should not make this exercise, what should be done is creating a new object
but ommiting the keys we don't want.

Hey good news, thats [omit](https://lodash.com/docs/4.17.15#omit), we could merge this with "pick" that only returns key you specified.

*Why not removing ?*
2 reasons:
- `bug prone`: objects are shared, all variables pointing to this object would get "removed" keys.
- `performances`: v8 and other javascript engine actually create optimized static structs for your objects. Those allow fast & compact allocation.
Now if you add a key, the handle that very well, they just create a new one with the new key added.
If you remove a key however they start thinking, wow we can't use our struct for that the shape has changed!, let's fallback to dynamic allocation. And now everything is very slow, your object will never get fast.

> *TL; DR:* Don't remove from object, re-create, it might not even use more memory in the end.
